docs/howto/system_testing.md

```diff
@@ -599,6 +600,83 @@ Finally, when you are done running all system tests, bring down the Elastic Stac
 elastic-package stack down
 ```
 
+Starting with [elastic-package version `v0.101.2`](https://github.com/elastic/elastic-package/releases/tag/v0.101.2),
```

Comment:
Include all this information about independent Elastic Agents under the section `Running system test`.

A new section has been added to explain how to run system tests using the Elastic Agent from the stack (as it was performed before).